apps/mull-api/src/database/factories/event.factory.ts

```diff
@@ -0,0 +1,20 @@
+import * as Faker from 'faker';
+import { define, factory } from 'typeorm-seeding';
+import { Event, Media, User } from '../../app/entities';
+
+define(Event, (faker: typeof Faker) => {
+  const event = new Event();
+
+  event.title = faker.lorem.lines();
+  event.description = faker.lorem.paragraph();
```

Comment:
You set limits for the `title` and `description` columns in `apps/mull-api/src/app/entities/event.entity.ts`. It might be wise to configure the calls to these methods so they don't go over.

Reply:
We were discussing this to maybe add a limit to the front end where the words would get cut off after a certain number of characters. Again we can discuss this quickly with the team

Reply:
I was talking about seeding. You set a larger limit for the columns. But to make sure stuff is resilient, you should check that the `faker.lorem` calls are configured right to not go over.

Also, some suggestions after looking at the docs: https://github.com/Marak/faker.js/blob/master/doc/faker.lorem.html

you should use `faker.lorem.words` for both of these, since it seems to offer the most control (you can choose the number of words).

I'd say put 5 words for `event.title` and something like 50 for `event.description`

Reply:
I don't think we should set a limit of the number of words for the description. I think we should use lorem.paragraph and give the users plenty of room to write what they want.

Reply:
I don't know if you understand what I'm trying to say.

The user isn't writing anything when you're seeding. I never said anything about `set a limit of the number of words for the description`. It's about making sure we don't over the limit we set for the column. I have no idea how much lorem.paragraph generates, since it's random. On the other hand, `lorem.words` is a lot easier to control. We can add more or less as we need.
![image](https://user-images.githubusercontent.com/23544999/105079461-749f7e00-5a5d-11eb-8615-32cc9c2890b2.png)
![image](https://user-images.githubusercontent.com/23544999/105079478-7ec17c80-5a5d-11eb-9209-9b556856acaf.png)

Reply:
but that's the thing I don't want to control that to a certain extent. But I do want it to be random. I know the user isn't writing anything but assume that it is a user that is writing and is getting an error saying that their description is too short.

Also if it does give a text too short error, it will still seed the previous entries

Reply:
I don't think the focus should be on the seeding. I think the focus should be the limit of characters in the column